doc/migrating-configs.md

```diff
@@ -2,544 +2,130 @@
 
 Occasionally, there are changes made to Ignition's configuration that break backward compatibility. While this is not a concern for running machines (since Ignition only runs one time during first boot), it is a concern for those who maintain configuration files. This document serves to detail each of the breaking changes and tries to provide some reasoning for the change. This does not cover all of the changes to the spec - just those that need to be considered when migrating from one version to the next.
 
-## From Version 2.2.0 to 2.3.0
+## From 2.x.0 to 2.3.0
 
-There are not any breaking changes between versions 2.2.0 and versions 2.3.0 of the configuration specification. Any valid 2.2.0 configuration can be updated to a 2.3.0 configuration by simply changing the version string in the config.
+Refer to this doc in the `spec2x` branch of this repository.
```

Comment:
We should link to it, at the very least.  If we want the doc to be rendered on a website somewhere, we should probably continue inlining the content.

Reply:
:+1: for linking. I think I'm fine with not inlining. If we render to an external site we can link to the coreos.com docs instead of the repo.